src/com/google/enterprise/adaptor/documentum/DocumentumAdaptor.java

```diff
@@ -609,11 +608,11 @@ private void validateModifiedDocumentsQuery(IDfSession session)
     try {
```

Comment:
Sorry, I wasn't paying enough attention. The try statement needs to be extended above the call to MessageFormat.format.